packages/block-library/src/audio/index.js

```diff
@@ -90,9 +90,25 @@ export const settings = {
 		const { autoplay, caption, loop, preload, src } = attributes;
 		return (
 			<figure>
-				<audio controls="controls" src={ src } autoPlay={ autoplay } loop={ loop } preload={ preload } />
-				{ ! RichText.isEmpty( caption ) && <RichText.Content tagName="figcaption" value={ caption } /> }
+				{ src && (
```

Comment:
Since the other attributes (autoplay, loop, preload) are also sourced from the markup, you'll probably lose their value if you tweak them and don't provider a source.

Reply:
It is not possible to set these attributes without a source. If there is no source file, you just see the placeholder in the post and no options in the sidebar

Reply:
Got it 👍 thanks for clarifying